### ISSUE TITLE
Mega EverDrive: Tanglewood

### ISSUE THREAD

**Issue by tjanas** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#173]
What is the explanation behind these two entries having different checksums? How are they different and why?

```
f843c8ea95d41f943777f9fa9a9ca8c360c26c5053f052eff8bd0d333ad9d440	Mega EverDrive/4 Homebrew/Tanglewood - Big Evil Corp.md	69a1f52ae4f881cab3663a0a0d5d38f2bc1aa2d5	49f7c30f65704aee46c1f6ac8953ede6	93db8aed
fcd9f37a0a283b0dc28d4059f0d5c8cb9b3911797ca0188c028ed80bd732b50e	Mega EverDrive/4 Homebrew/Tanglewood - Big Evil Corp (PD).md	128b547cad971434e6dd815d1c65a268d826d831	b684990797358eb247bfc466ea606292	1a91c419
```

The copy I have has a checksum of 1a91c419.

**Comment by frederic-mahe** [comment]
I only have a demo of Tanglewood (v0.9.37) with a different SHA256 (5d64ea0b5abfa5eadfc5800fbdff3a1ee647039bc506d2240709554a7cc96d89). If you open the ROM with a hexeditor you can get the version number.

**Comment by maniac79** [comment]
This is only an assumption but I think one of the hashes may be a version that is distributed via STEAM?
According to this site: https://segaretro.org/Tanglewood#ROM_dump_status
there is only 1 known dump of the cartridge and that matches with the 2nd entry of the SMDB.

I suggest removing the 1st entry until it's known what it is.
I'll submit a PR soon which removes this entry.

**Comment by frederic-mahe** [comment]
fixed in a54ab0059d33ddc730b79b4d42833681bd166a8c , thanks @maniac79
